release/README.txt

```diff
@@ -0,0 +1,97 @@
+# Dark Souls III Archipelago Randomizer 3.0.0-beta.8
+
+This is a preview release of a new architecture for randomizing Dark Souls III for the Archipelago multiworld randomizer. It has a number of major feature improvements over the old 2.x.x line, including:
+
+* Support for randomizing all item locations, not just unique items, without needing progressive pickup lists.
+
+* Support for randomizing items in shops, starting loadouts, Path of the Dragon, and more.
+
+* Built-in integration with the enemy randomizer, including consistent seeding for races and configuration via the web interface and the standard Archipelago YAML file.
+
+* Support for the latest patch for Dark Souls III, 1.15.2. Note that older patches are currently *not* supported, although if ModEngine2 ever adds support they should work fine.
+
+* Optional smooth distribution for upgrade items, upgraded weapons, and soul items so you're more likely to see weaker items earlier and more powerful items later.
+
+* More detailed location names that indicate where a location is, not just what it replaces.
+
+* Other players' item names are visible in DS3.
+
+* If you pick up items while offline, they'll still send once you reconnect.
+ 
+* Many more improvements.
+
+## Generating a Multiworld
+
+To use this preview, you must generate your multiworld with my fork of Archipelago, located in the `archipelago` directory of the release below. See [the "Running From Source" documentation] for more details. Once you've generated a ZIP file, upload it to https://archipelago.gg/uploads and you're good to go. (Unfortunately running from an `.apworld` file won't work until [this pull request] is merged and released.)
+
+[the "Running From Source" documentation]: https://github.com/nex3/Archipelago/blob/ds3-3.0.0-beta.8/docs/running%20from%20source.md
+[this pull request]: https://github.com/ArchipelagoMW/Archipelago/pull/2403
+
+Note that not all YAML fields for Dark Souls III's 2.x.x Archipelago randomizer are supported as-is, and some new fields are available. Consider generating a new YAML configuration using the `Dark Souls III Options Template.yaml` file included in this release.
+
+The following options have been removed:
+
+* `enable_boss_locations` is now controlled by the `soul_locations` option.
+
+* `enable_progressive_locations` was removed because all locations are now individually randomized rather than replaced with a progressive list.
+
+* `pool_type` has been removed. Since there are no longer any non-randomized items in randomized categories, there's not a meaningful distinction between "shuffle" and "various" mode.
+
+In addition, the following options have changed:
+
+* `enable_*_locations` options have all been deprecated. Instead, there are a number of named location groups that you can add to the `exclude_locations` option to prevent them from containing important items. These location groups are:
```

Comment:
Just like in the AP server, I suggest to mark those locations as removed while keeping the explainations on how to setup "exclude_locations" properly